src/mrunal/student.java

```diff
@@ -0,0 +1,42 @@
+	/*Assignment : 48
+	Student class having attribute -> studentId, studentName, studentStd.
+	A) Use comparable to sort all students based on studentName.*/
+
+package mrunal;
+
+public class student implements Comparable {
+
+		private String studentName;
+		private int studentId;
+		private int studentStd;
+
+		public student(String studentName, int studentId, int studentStd) {
+			this.studentName = studentName;
+			this.studentId = studentId;
+			this.studentStd = studentStd;
+		}
+
+		private String getstudentName() {
+			return studentName;
+		}
+		
+		private int getstudentID() {
+			return studentId;
+		}
+
+		private int getstudentStd() {
+			return studentStd;
+		}
+		
+		public int compareTo(Object arg0) {
```

Comment:
If you make Comparable<Student> then you can write Student instead of Object, then you don't require to type cast to student